converter/src/test/java/gov/cms/qpp/conversion/validate/ClinicalDocumentValidatorTest.java

```diff
@@ -229,14 +229,15 @@ void testClinicalDocumentValidationParsesMultipleErrors() {
 
 		List<Detail> errors = getErrors(allErrors);
 
-		assertWithMessage("Must have 2 errors")
-				.that(errors).hasSize(2);
+		assertWithMessage("Must have 3 errors")
```

Comment:
@saquino0827 Not sure about this change, I took my best shot at this but not quite sure... can you verify what I have is correct?

Reply:
Gotcha, I'll take a look.

Reply:
Looks fine! Nice work Shad!